src/librustc/diagnostics.rs

```diff
@@ -1941,6 +1941,53 @@ you want. Example:
 ```
 "##,
 
+E0492: r##"
+A constant borrow which contains interior mutability was attempted. Erroneous
+code example:
+
+```
+#![feature(const_fn)]
+
+use std::cell::UnsafeCell;
+
+const A: UnsafeCell<usize> = UnsafeCell::new(1);
```

Comment:
Use Cell or RefCell here, not UnsafeCell. UnsafeCell is something people should be using very rarely.